int64 pin
int64 value
---
bool success
string message
